screenpy_selenium/__init__.py

```diff
@@ -16,9 +16,30 @@
 :license: MIT, see LICENSE for more details.
 """
 
+import screenpy
+from screenpy import *  # noqa
 
-from .target import Target
+from . import abilities, actions, questions, resolutions
+from .abilities import *  # noqa
+from .actions import *  # noqa
+from .exceptions import BrowsingError, TargetingError  # noqa
+from .protocols import Chainable  # noqa
+from .questions import *  # noqa
+from .resolutions import *  # noqa
+from .target import Target  # noqa
 
-__all__ = [
-    "Target",
-]
+__all__ = list(
+    set(
+        [
+            "Target",
+            "TargetingError",
+            "BrowsingError",
+            "Chainable",
+        ]
+        + screenpy.__all__
+        + abilities.__all__
+        + actions.__all__
+        + questions.__all__
+        + resolutions.__all__
+    )
+)
```

Comment:
Using set is to weed out duplicate entries (Pause is overwritten).  It's unclear if this really needs to be done but felt like the right thing to do.  Unfortunately flake8 doesn't have the ability to recognize that imports are being used when this is done, thus the additional `# noqa` everywhere.